docker/qgisserver/tests/functional/conftest.py

```diff
@@ -97,3 +97,23 @@ def multiple_ogc_server_env():
     os.environ["GEOMAPFISH_ACCESSCONTROL_CONFIG"] = "/etc/qgisserver/multiple_ogc_server.yaml"
     yield
     del os.environ["GEOMAPFISH_ACCESSCONTROL_CONFIG"]
+
+
+@pytest.fixture(scope="class")
+def auto_multi_ogc_server_env():
+    os.environ["GEOMAPFISH_ACCESSCONTROL_BASE_URL"] = "http://qgis"
+    old = os.environ["QGIS_PROJECT_FILE"]
+    del os.environ["QGIS_PROJECT_FILE"]
+    yield
+    del os.environ["GEOMAPFISH_ACCESSCONTROL_BASE_URL"]
+    os.environ["QGIS_PROJECT_FILE"] = old
```

Comment:
Could be:
```
def auto_multi_ogc_server_env()
    with patch.dict(
        "geomapfish_qgisserver.accesscontrol.os.environ",
        {
            "GEOMAPFISH_ACCESSCONTROL_BASE_URL": "http://qgis",
            ...
        }
    ):
        yield
```
See: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch.dict

Reply:
Thanks :-)